src/platforms/apollo/schemas.ts

```diff
@@ -295,6 +300,14 @@ export const apolloCallDisposition = z.object({
 });
 export type ApolloCallDisposition = z.infer<typeof apolloCallDisposition>;
 
+export const apolloCallPurpose = z.object({
```

Comment:
Is this supposed to be used in the call create action?

Reply:
Ah no, this is used in apolloBootstrappedDataSchema which is returned in `calls/details`